### PR TITLE
Fix docker-compose to read DB_HOST or use fallback 127.0.0.1

### LINKED ISSUE #34: Resolve Suggestions According to ToB Report 6, MAY 2024
### Description

3\. Missing certificate validation (Med)
4\. Synchronous function calls inside asynchronous functions (Info)
5\. Hardcoded credentials (Low)
6\. Use of outdated libraries (Info)
7\. Stack traces in Express are not disabled (Info)
8\. Docker Compose ports exposed on all interfaces (Info)

For Item 7, this can be resolved by adding flag in AWS deployment script, which is not included in this Repo

### PR DESCRIPTION
**Pull request by Phanco**
### What was the problem?

This PR resolves #34, Item 8

### How was it solved?
`DB_HOST` is passed to docker file, developer can decide to use `127.0.0.1` (more restriction) or `0.0.0.0` (expose to public)

### How was it tested?
Tested and verified data in Docker dashboard

### CODE REVIEW

File: packages/claim-backend/docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     image: postgres
     restart: always
     ports:
-      - '${DB_PORT}:5432'
+      - '${DB_HOST:-127.0.0.1}:${DB_PORT}:5432'
```
Comment (matjazv) [inline packages/claim-backend/docker-compose.yml]: Is minus sign `-` intentionally before `127.0.0.1`?
It's not before `5432`.

Comment (Phanco) [reply packages/claim-backend/docker-compose.yml]: It is intentional of implementing fallback param [ref](https://stackoverflow.com/questions/70772517/docker-compose-env-default-with-fallback)

**Review by matjazv — APPROVED** [review]

File: packages/claim-backend/docker-compose.yml (hunk 1 above)
Comment (sameersubudhi) [inline packages/claim-backend/docker-compose.yml]: Please assign the default port as well, else docker assigns random ports when `DB_PORT` environment variable is not specified.
```suggestion
      - '${DB_HOST:-127.0.0.1}:${DB_PORT:-5432}:5432'
```

Comment (Phanco) [reply packages/claim-backend/docker-compose.yml]: Updated 👍

**Review by sameersubudhi — APPROVED** [review]
🚀

**Pull request merged**